cli/src/pcluster/validators/fsx_validators.py

```diff
@@ -222,3 +222,16 @@ def _validate(self, auto_import_policy, import_path):
             bucket = get_bucket_name_from_s3_url(import_path)
             if AWSApi.instance().s3.get_bucket_region(bucket) != get_region():
                 self._add_failure("FSx auto import is not supported for cross-region buckets.", FailureLevel.ERROR)
+
+
+class FsxDescribeVolumesValidator(Validator):
+    """Backup id validator."""
+
+    def _validate(self, volume_ids):
+        try:
+            AWSApi.instance().fsx.describe_volumes(volume_ids)
```

Comment:
I assume that you're adding this AWS call to trigger an interaction with FSx, but our goal here is to test the existent interaction.

Reply:
Yes, however mocking the existent interaction even within a unit test seems to be much more involved than taking this approach.

Reply:
Ok, that makes sense.